### PR TITLE
Add suffixes to typos from "pe" to "po"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -38049,8 +38049,22 @@ peaseful->peaceful
 peasefully->peacefully
 peases->pieces, pleases, leases,
 peasing->piecing, pleasing, leasing,
+pecent->percent, recent, decent,
 pecentage->percentage
+pecentages->percentages
+pecentile->percentile
+pecentiles->percentiles
+pecents->percents
+pecific->specific, Pacific,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38058]:
```suggestion
pecific->specific, pacific,
```
Adjective `pacific` as `peaceful`. In such cases, we drop the uppercase, and codespell will fix on its own as follows:
```
pecific → pacific
Pecific → Pacific
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -38049,8 +38049,22 @@ peaseful->peaceful
...
+pecifically->specifically
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38059]: Adverb `pacifically` is rare but does exist.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -38066,6 +38080,9 @@ peepels->peoples
 peerowet->pirouette
 peerowetes->pirouettes
 peerowets->pirouettes
+pefect->perfect, prefect,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38083]: Perhaps `defect` too? I can find (few) occurrences of `pefects->defects`.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -38430,6 +38471,16 @@ perseeded->preceded
 perseedes->precedes
 perseeding->preceding
 perseeds->precedes
+persent->present, percent,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38474]: Perhaps `persistent` too?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -38708,8 +38774,11 @@ piplelines->pipelines
 pipline->pipeline
 piplines->pipelines
 pirric->Pyrrhic
+pithon->python
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38777]: Any occurrences of this one?
```suggestion
pithon->python, piton,
```

Comment (int-y1) [reply codespell_lib/data/dictionary.txt:38777]: This exists, but is rare.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**